src/main/java/com/puppycrawl/tools/checkstyle/api/FileContents.java

```diff
@@ -328,9 +329,11 @@ private boolean hasIntersectionWithSingleLineComment(int startLineNo, int startC
      * Checks if the current file is a package-info.java file.
      *
      * @return true if the package file.
+     * @deprecated use {@link CheckUtil#isPackageInfo(String)}
+     *              and {@link AbstractCheck#getFilePath()} instead.
```

Comment:
It seems to me `CheckUtil#isPackageInfo` is enough to replace `inPackageInfo`.

Reply:
I think no, because getFileContents is deprecated as well, so in my opinion it is better to note here method for getting file name

Reply:
changed